### DISCUSSION TITLE
Game screen layout

### DISCUSSION THREAD

**Discussion by LeoNunk** [meoowe/ClickyCat#2] [General]
I think that the ginger cat should be in the centre of the game screen, with the wooden house to the right. I have created flat ground pixel art for the cat and house to stand on top of. (I create all the pixel art featured in Clicky Cat)

**Comment by meoowe** [comment]
Ok, great idea, lets do it. Only thing i would say is the cat will be climbing a mountain, so it won't be possible to keep it next to the housed.

**Comment by meoowe** [comment]
should i use image 1, 2 or 3 for the flat grass

**Reply by LeoNunk** [reply]
Alternate between all 3 to give it a bit of variety.

**Reply by meoowe** [reply]
Ok!

**Comment by meoowe** [comment]
Here  is the curve for the hill
![Screenshot 2023-11-24 200714](https://github.com/meoowe/ClickyCat/assets/96916064/1455a1b8-0dbe-47c4-93f8-3e0d45a43498)

**Reply by meoowe** [reply]
we need a bigger hill

https://github.com/meoowe/ClickyCat/assets/96916064/ffe62982-7e34-4c42-b1dc-7d40437c3379

**Comment by meoowe** [comment]
What do you think?
![Screenshot 2023-11-24 225628](https://github.com/meoowe/ClickyCat/assets/96916064/d87cd36e-b98c-48f3-a5ca-3a2dd038cfb7)
